x/fungible/keeper/zrc20_methods.go

```diff
@@ -14,13 +14,179 @@ import (
 )
 
 const (
+	setName      = "setName"
+	setSymbol    = "setSymbol"
+	name         = "name"
+	symbol       = "symbol"
 	allowance    = "allowance"
 	balanceOf    = "balanceOf"
 	totalSupply  = "totalSupply"
 	transfer     = "transfer"
 	transferFrom = "transferFrom"
 )
 
+// ZRC20SetName updates the name of a ZRC20 token
+func (k Keeper) ZRC20SetName(
```

Comment:
The naming is a bit confusing IMO
```suggestion
func (k Keeper) SetZRC20Name(
```

Reply:
Also, do we need 2 separate functions? `ZRC20SetSymbol`

Reply:
I follow the convention in the file for other methods `<ContractName><MethodName>`